### ISSUE TITLE
Port names only appear in the service.yaml file, not in deployment.yaml

### ISSUE THREAD

**Issue by jaydeboer** [prom3theu5/aspirational-manifests#213]
I really appreciate all your work. Thank you!

### 🔥 Bug Description
When using named ports in a `.WithEndpoint` in the Aspire project, the name only follows the port to the `service.yaml` file.  The name is missing in the `deployment.yaml` file.

### 🔍 Steps to Reproduce the Bug
1. Add named endpoint to a project
   ```csharp
   builder.AddProject<Bob>("bob").WithEndpoint(8558, scheme: "TCP", name:"management", isProxied:false);
   ```
2. Run `aspirate generate`
3. `service.yaml` has the name
   ```yaml
     - name: management
       port: 8558
       targetPort: 8558
   ```
5. `deployment.yaml` has the port but no name
   ```yaml
     ports: 
     - containerPort: 8558
   ```

I would have expected:

```yaml
  ports: 
   - containerPort: 8558
     name: management
```

### Disclaimer
I am new to Aspire, Aspirate, and k8s, so it is entirely possible I am barking up the wrong tree.  I am following a guide on how to deploy Akka.NET clusters to k8s, and they use a name port to do auto discovery of other nodes in the Akka.NET cluster.  I've written some `Kustomize` files to augment the port with the name and it makes it work.   It would be just nice if it could be generated out of the box. :)

**Comment by prom3theu5** [comment]
Thanks for reporting - I'll add tomorrow :)

**Comment by jaydeboer** [comment]
Fantastic!  Thank you.

**Comment by prom3theu5** [comment]
fixed in version 8.0.3

**Comment by jaydeboer** [comment]
Confirmed working!  Thanks so much!
